Add .bit(dotbit) method json file

zgayjjf opened the pull request:
After chatting with @hax, I have polished the spec. Please take another look.

### DID Method Registration

As a DID method registrant, I have ensured that my DID method registration complies with the following statements:

- [x] The DID Method specification [defines the DID Method Syntax](https://w3c.github.io/did-core/#method-syntax).
- [x] The DID Method specification [defines the Create, Read, Update, and Deactivate DID Method Operations](https://w3c.github.io/did-core/#method-operations).
- [x] The DID Method specification [contains a Security Considerations section](https://w3c.github.io/did-core/#security-requirements).
- [x] The DID Method specification [contains a Privacy Considerations section](https://w3c.github.io/did-core/#privacy-requirements).
- [x] The JSON file I am submitting has [passed all automated validation tests below](#partial-pull-merging).
- [x] The JSON file contains a `contactEmail` address [OPTIONAL].
- [x] The JSON file contains a `verifiableDataRegistry` entry [OPTIONAL].

OR13 commented on methods/dotbit.json:
```diff
@@ -0,0 +1,9 @@
+  "verifiableDataRegistry": "Nervos CKB",
+  "contactName": "dotbit",
+  "contactEmail": "jeff@did.id",
+  "contactWebsite": "https://did.id/",
```
clever :)

zgayjjf replied:
We have a even cleverer domain :P

OR13 commented on methods/dotbit.json:
```diff
@@ -0,0 +1,9 @@
+  "contactName": "dotbit",
+  "contactEmail": "jeff@did.id",
+  "contactWebsite": "https://did.id/",
+  "specification": "https://github.com/zgayjjf/dotbit-did-spec"
```
method does not describe deactivate, privacy and security considerations section could use more text.

zgayjjf commented:
> describe deactivate, add privacy considerations

Hi, @OR13 

We have added sections on privacy consideration and deactivation, and the section of security consideration was also updated.

You can take a look here:

https://github.com/zgayjjf/dotbit-did-spec/commit/b63873be2b892b9179d564a5f0fe7fd690fc74a5

zgayjjf commented:
Hi @OR13,
Can you take a look at this PR. I have resolved the issues.
